### ISSUE TITLE
排名轮播表加载不全

### ISSUE THREAD

**Issue by yangyanju** [DataV-Team/DataV#153]
dv-scroll-ranking-board排名轮播表，整页滚动的时候加载不全，经常出现只加载一条，然后过好一会儿才出现其他的内容，这个有解决办法吗？

**Comment by jiaming743** [comment]
有没有复现例子

**Comment by yangyanju** [comment]
http://datav.jiaminghi.com/guide/scrollRankingBoard.html#%E5%8D%95%E6%9D%A1%E6%BB%9A%E5%8A%A8

就这个整页滚动的例子也是加载不全的
![111](https://user-images.githubusercontent.com/7056767/97548236-d4abf880-1a09-11eb-8961-d1d517ccb39e.png)
![Uploading 222.png…]()

**Comment by yangyanju** [comment]
![222](https://user-images.githubusercontent.com/7056767/97548389-08871e00-1a0a-11eb-9a0e-4223b90a6073.png)

**Comment by mike652638** [comment]
对, 遇到同样问题, 官网文档示例中的整页滚动也是明显存在这个问题的(整页滚动时先只出现最上方一条记录, 隔几秒再出现整页的), 特意来issue看下是否有过反馈http://datav.jiaminghi.com/guide/scrollRankingBoard.html#%E5%8D%95%E6%9D%A1%E6%BB%9A%E5%8A%A8
http://datav-react.jiaminghi.com/guide/scrollRankingBoard.html#%E5%8D%95%E6%9D%A1%E6%BB%9A%E5%8A%A8

**Comment by zyc95** [comment]
+1同遇到相同问题，请问有办法解决么？

**Comment by zyc95** [comment]
而且还遇到一个问题，如果单条滚动的话，一旦整个页面超过一屏可以滚动的时候，把页面滚动到底部，页面会自己跑上去。
